integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImage.java

```diff
@@ -0,0 +1,227 @@
+      // domain = new Domain(domainMap, true, false);
+      domain.verifyDomainCreated();
+      testCompletedSuccessfully = true;
+    } finally {
+      if (domain != null && (JENKINS || testCompletedSuccessfully)) {
+        TestUtils.deleteWeblogicDomainResources(domain.getDomainUid());
+      }
+    }
+
+    LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
+  }
+
+  /**
+   * Create a domain using model in image and having configmap in the domain.yaml
+   * before deploying the domain. After deploying the domain crd, create a new
+   * config map and update the domain crd to new config map and change domain
+   * restartVersion to reload the model, generate new config and initiate a
+   * rolling restart.
+   * @throws Exception exception
+   */
+  @Test
+  public void testMiiWithConfigMapBothBeforeAndAfterDeployingDomain() throws Exception {
+    Assumptions.assumeTrue(QUICKTEST);
+    String testMethodName = new Object() {
+    }.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    LoggerHelper.getLocal().log(Level.INFO,
+        "Creating Domain & waiting for the script to complete execution");
+    Domain domain = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+
+      domain = createMIIDomainWithConfigMap(testClassName, domainNS, "./model.wls.yaml",
+          "./model.empty.properties", "./model.properties", "WLS");
+      Map<String, Object> domainMap = domain.getDomainMap();
+      //ToDo: access MS using port given in the configmap props
+
+      // config map after deploying domain crd
+      String cmName = domainMap.get("domainUID") + "-mii-config-map2";
+      String cmFile = getResultDir() + "/samples/model-in-image/model.cm.properties";
+      TestUtils.createConfigMap(cmName, cmFile, domainNS,
+          " weblogic.domainUID=" + domainMap.get("domainUID"));
+
+      // update domain yaml with new config map, restartVersion and
+      // apply the domain yaml, verify domain restarted
+      modifyDomainYamlWithNewConfigMapAndDomainRestartVersion(
+          cmName, domain);
+      LoggerHelper.getLocal().log(Level.INFO, "Verifying if the domain is restarted");
+      domain.verifyDomainRestarted();
+      //ToDo: access MS using port given in the new configmap props
+      testCompletedSuccessfully = true;
+    } finally {
+      if (domain != null && (JENKINS || testCompletedSuccessfully)) {
+        TestUtils.deleteWeblogicDomainResources(domain.getDomainUid());
+      }
+    }
+
+    LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
+  }
+
+  /**
+   * Negative test. overridesConfigMap should fail with model in image domain.
+   * Create a domain using model in image and deploy the domain.
+   * After deploying the domain, update domain crd with overrides configmap and
+   * apply the crd.
+   * Verify that the domain deployment fails with proper error in the operator log.
+   *
+   * @throws Exception exception
+   */
+  //@Test
```

Comment:
Why is `@Test` commented out?

Reply:
@vanajamukkara This one is for you.

Reply:
The test was not fully developed and tested.

Reply:
Let's remove it then

Reply:
ok